### PR TITLE
Finally, a use for Peas.

### PR DESCRIPTION
**Pull request by TheObserver-sys**
## About The Pull Request
Salad of Eden - Requires a World Pea Cluster, and one sprig of each Ambrosia mutation. Grants Earthsblood, Omnizine, and Happiness as well as nutriment. High effort, high reward.

Fish Fry - Requires carp fillet, peas, and corn. Generally loved by everyone.

Gumbo - Requires boiled rice, peas, a chili pepper, and a cutlet of meat. Grants capsaicin.

Pea Soup - Requires water, 2 peas, a parsnip, and a carrot. Grants occuline in small measure.

Sweet Pea Donut/Jelly Donut - Requires you to juice a Laughin' Pea to produce Laughin' Syrup. Donuts give a slight amount of laughter.

Big ups to Stewydeadmike for the wonderful work on sprites!
It's about time I actually got this shit over here, innit? _More to come. Stay tuned._

## Why It's Good For The Game
More choice is good. Also, having actual foodstuffs attached to your plants gives them more reason to exist outside specific chem traits or medicines, eh?

## Changelog
:cl: The0bserver and Stewydeadmike
add: Hey, there's a bit of dust in this recipe book! Recipes using peas? How many recipes does this book even have?
add: Adds 6 new food items, and 1 new consumable reagent using all forms of the recently discovered peas. Ask your local botanist and chef for them today!
/:cl:

### CODE REVIEW

**Comment by ADepressedPyroshark** [comment]
When are we getting the pea shooter

**Comment by TheObserver-sys** [comment]
I'm working on the base version, which acts as a stamina damage pistol, expect to see that sooner than not, hopefuly.

**Comment by ADepressedPyroshark** [comment]
> 
> 
> I'm working on the base version, which acts as a stamina damage pistol, expect to see that sooner than not, hopefully.

Coggers

**Review by triplezeta — APPROVED** [review]
Everything seems solid and the sprites are up to my standards

good shit 10/10

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -54,6 +54,15 @@
 	tastes = list("fish" = 1, "chips" = 1)
 	foodtype = MEAT | VEGETABLES | FRIED
 
+/obj/item/reagent_containers/food/snacks/fishfry
```
Comment (Maldaris) [inline code/modules/food_and_drinks/food/snacks_meat.dm:57]: New code should exist in the `waspstation/` modularization folder if at all possible. It looks like that's the case here. You can create the folder structure/file using the same path here.

**Review by Maldaris — CHANGES_REQUESTED** [review]
Looks like most of this code needs to be moved into the `waspstation/` folder, otherwise this looks fine.

File: code/modules/hydroponics/grown/peas.dm [hunk 2]
```diff
@@ -60,6 +60,7 @@
 	filling_color = "#ee7bee"
 	bitesize_mod = 2
 	foodtype = VEGETABLES
+	juice_results = list(/datum/reagent/consumable/laughsyrup = 0)
```
Comment (Maldaris) [inline code/modules/hydroponics/grown/peas.dm:63]: Needs a modularization comment

**Review by Maldaris — CHANGES_REQUESTED** [review]
Single modularization comment, otherwise this looks great.

File: code/modules/hydroponics/grown/peas.dm (hunk 2 above)
Comment (Superyodeler) [reply code/modules/hydroponics/grown/peas.dm:63]: Could possibly be removed, since it's adding 0 of that reagent.

Comment (Superyodeler) [reply code/modules/hydroponics/grown/peas.dm:63]: Modularization comment would look like:
`// Waspstation edit - Pea recipies`

Comment (triplezeta) [reply code/modules/hydroponics/grown/peas.dm:64]: weirdly enough I think that's how juice_results is supposed to be 
image

Comment (Superyodeler) [reply code/modules/hydroponics/grown/peas.dm:64]: I hate to say it, but I've seen crazier.
Just needs this in that case:
```suggestion
	juice_results = list(/datum/reagent/consumable/laughsyrup = 0)		// Waspstation edit - Pea recipes
```

Comment (triplezeta) [reply code/modules/hydroponics/grown/peas.dm:64]: image
 bruh??

Comment (Superyodeler) [reply code/modules/hydroponics/grown/peas.dm:64]: Whoops, didn't see that

**Review by Maldaris — APPROVED** [review]

**Pull request merged**